setup_default_backend.sh

```diff
@@ -36,10 +37,7 @@ function install_backend_system_dep {
     log "install base debian-packaged-dep for backend..."
     $SUDO apt update
     PACKAGE_LIST='python3 python3-venv python3-dev python3-pip redis-server zlib1g-dev libjpeg-dev imagemagick libmagickwand-dev libpq-dev ghostscript libfile-mimeinfo-perl poppler-utils libimage-exiftool-perl qpdf libldap2-dev libsasl2-dev libreoffice inkscape ufraw-batch ffmpeg'
-    for PACKAGE in $PACKAGE_LIST
-    do
-        $SUDO apt install -y $PACKAGE && loggood "$PACKAGE correctly installed" || logerror "failed to install $PACKAGE"
-    done
```

Comment:
I remember why we have used this. On some installation we have seen packages error and its not easy to find which packages is not installed correctly in all log.

Reply:
apt should print which packages are problematic, no? Do you think this is still problematic?

Reply:
I need to search in history of this fix to see if an issue exist (and explain why we have used this).

Reply:
Did you get any information on this ?
If not, we may put this in 3.2.X and solve this later.

Reply:
after discussion, Phillippe agrees with this design